### ISSUE TITLE
Inlay Hints got too bright.

### ISSUE THREAD

**Issue by vixalie** [Monokai/monokai-pro-vscode#290]
When enabled "Inlay Hints" in Typescript plugin, those hints is too bright. Suggest make them darker.

<img width="668" alt="截屏2022-02-25 上午9 04 41" src="https://user-images.githubusercontent.com/1289513/155633977-87b8d781-65e9-431d-99ca-804d51d6d666.png">

**Comment by robertwayne** [comment]
Just an FYI for anyone who wants to fix this themselves, add this line to your global settings.json _(or .vscode/settings.json)_ file:

```json
"workbench.colorCustomizations": {
    "editorInlayHint.background": "#2a2b2e",
    "editorInlayHint.foreground": "#b3aeae"
},
```

**Comment by Monokai** [comment]
fixed in 1.1.21
